### PR TITLE
Minor tidying of mtools page

### PR DESCRIPTION
**Pull request by hflynn**
While I was checking the downloads pages for 'legacy' sections, I noticed a couple of very minor inconsistencies.

### CODE REVIEW

File: mtools_downloads.html [hunk 1]
```diff
@@ -40,18 +40,21 @@
 
 <h2>OMERO.mtools @VERSION@ Downloads</h2>
 
+<p>
+  <strong><a href="#mtools">mtools</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="#mcr">MCR</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="#previous">Previous versions</a> </strong></p>
```
Comment (sbesson) [inline mtools_downloads.html]: Do we want mtools or rather `OMERO.mtools` here?

**Comment by hflynn** [comment]
I wasn't sure which to pick, I can change it to OMERO.mtools if you think that's better, I guess repetitive headings are better than confusing ones.

**Comment by sbesson** [comment]
Looks good. Let's go for that and wait for future releases to adjust if needed. Can you rebase onto `develop` ?

**Comment by hflynn** [comment]
--rebased-to #92

**Pull request merged**